elixir/apps/domain/lib/domain/auth/provider/changeset.ex

```diff
@@ -85,6 +85,10 @@ defmodule Domain.Auth.Provider.Changeset do
       name: :auth_providers_account_id_oidc_adapter_index,
       message: "this provider is already connected"
     )
+    |> unique_constraint(:base,
+      name: :unique_account_adapter_index,
+      message: "only of this adapter type may be enabled per account"
```

Comment:
Is this what you meant here?

```suggestion
      message: "only one of this adapter type may be enabled per account"
```

Reply:
👍